packages/@react-aria/calendar/stories/useCalendar.stories.tsx

```diff
@@ -36,3 +36,13 @@ export const Months1 = {
   render: () => <Example visibleDuration={{months: 1}} />,
   name: 'months: 1'
 };
+
+export const Months2 = {
+  render: () => <Example visibleDuration={{months: 2}} />,
```

Comment:
looks pretty good, only issue i found was that the example only renders 1 month here, but it does skip by 2 when hitting 'next'